src/gui/guiEngine.cpp

```diff
@@ -297,10 +297,14 @@ void GUIEngine::run()
 
 		driver->endScene();
 
+		IrrlichtDevice *device = RenderingEngine::get_raw_device();
+		u32 frametime_min = 1000 / (device->isWindowFocused()
```

Comment:
```suggestion
		u32 frametime_min = 1000 / (device->isWindowFocused() || g_menumgr.pausesGame()
```
If this works

Reply:
I think this is the main menu; this cannot be paused.